api-connection.html

```diff
@@ -100,7 +100,203 @@ <h2 id="function_newConnection">new Connection(config)</h2>
       <dd>
         Type of the authentication method, valid types are <code>default</code>, <code>ntlm</code>,
         <code>azure-active-directory-password</code>, <code>azure-active-directory-access-token</code>, 
-        <code>azure-active-directory-msi-vm</code>, or <code>azure-active-directory-msi-app-service</code> 
+        <code>azure-active-directory-msi-vm</code>, <code>azure-active-directory-service-principal-secret</code>,
+        or <code>azure-active-directory-msi-app-service</code> 
+        <ul>
+          <li><code>azure-active-directory-password</code></li>
+{% highlight javascript %}
+"authentication": {
+  ...,
+  "options": {
+    "userName": value;
+    "password": value;
+    "domain": value; (Optional)
```

Comment:
Should these be commas instead of semicolons?
```suggestion
  "options": {
    "userName": value,
    "password": value,
    "domain": value (Optional)
```

Reply:
Will fix this. Should not have missed this. By the way, do not think the double quotes are needed here as where?

Reply:
Quotes for the key is optional for JS objects so it's not necessary

Reply:
Agreed! Just checked the whole doc, seems all the other similar places keep the quotes, so I guess we could keep them here for now.